### DISCUSSION TITLE
Communication channel for the project

### DISCUSSION THREAD

**Discussion by RajGM** [hyperledger/cactus#2177] [General]
Is there any communication channel apart from Github to engage with the community?

**Comment by Yogesh01000100** [comment]
are there any video tutorials available for cactus integration with hyperledger fabric??

**Comment by petermetz** [comment]
> Is there any communication channel apart from Github to engage with the community?

@RajGM Sorry for the slow response! You can also use the Discord channel(s) [1] and the daily pair programming calls[2]!

[1] https://discord.gg/invite/hyperledger
[2] https://wiki.hyperledger.org/display/cactus/Daily+Pair+Programming+Calls

**Reply by RajGM** [reply]
Thanks for the response, @petermetz

**Comment by petermetz** [comment]
> are there any video tutorials available for cactus integration with hyperledger fabric??

@Yogesh01000100  What comes to mind is last year's workshop (there's a lot more information in there not just Fabric integration specific things though so you might want to go through the table of contents to see which section is most relevant)
https://www.youtube.com/watch?v=TM-dnP2yzRM
